packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -107,7 +107,7 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
     // Let child element (e.g. menu button) handle the event if the Alt key is pressed.
     // Keyboard events bubble through portals. Don't handle keyboard events
     // for elements outside the collection (e.g. menus).
-    if (e.altKey || !ref.current.contains(e.target as HTMLElement)) {
+    if ((e.altKey && !e.ctrlKey) || !ref.current.contains(e.target as HTMLElement)) {
```

Comment:
This addition is supposed to prevent ctrl + alt (aka control + option on Mac) from being handled by the child element right? Right now in https://reactspectrum.blob.core.windows.net/reactspectrum/83a514c745dcc6475b057e38ac23c3292b8c8770/storybook/index.html?path=/story/tableview--crud ctrl + alt + arrow down still opens the menu.

Reply:
that story isn't a replacement behavior, so it shouldn't need this anyways?